packages/design-system/src/components/tooltip/index.js

```diff
@@ -133,6 +134,17 @@ function Tooltip({
 
     setArrowDelta(delta);
   }, []);
+  // When near the bottom of the viewport and the tooltip is placed on the bottom we want to force the tooltip to the top as to not
+  // cutoff the contents of the tooltip.
+  const positionPlacement = useCallback(() => {
+    const tooltipElBoundingBox = tooltipRef.current?.getBoundingClientRect();
```

Comment:
Chatted with Brooke about this but putting here for visibility:

Only thing I would do here is maybe put it inside a check like
```
if (placement.startsWith('bottom') {
...
}
```

That way we don't run that every time if we don't need to.